File: py/hello.py
import skilstak.colors as c

def print_plain(message):
    """prints <message> in plain text>"""
    print(message)

def print_color(message):
    """Prints hello in a random color each time"""
    print(c.rc() + message + "!" + c.reset)

def print_nyan(message):
    """Prints <message> forever"""
    while True:
        print(c.rc() + message , end=" ")

File: go/src/hello/hello.go
package hello

import (
	"fmt"
	c "github.com/skilstak/go/colors"
	"os"
	//i "github.com/skilstak/go/input"
)

func printMulti(message string) {
	fmt.Println(c.Clear + c.Multi(message) + c.X)
}
